src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -838,11 +838,22 @@ private function registerAnnotationsConfiguration(array $config, ContainerBuilde
 
     private function registerPropertyAccessConfiguration(array $config, ContainerBuilder $container)
     {
-        $container
+        $definition = $container
             ->getDefinition('property_accessor')
             ->replaceArgument(0, $config['magic_call'])
             ->replaceArgument(1, $config['throw_exception_on_invalid_index'])
         ;
+
+        if (isset($config['cache']) && $config['cache']) {
+            $container->setParameter(
+                'property_accessor.cache.prefix',
```

Comment:
This feature (changing the namespace based on the `kernel.root_dir` to automatically invalidate the cache on deployment) should be provided by the `doctrine-cache-bundle` generically. It's not the task of framework-bundle to configure the doctrine-cache. And this configuration is only used in certain circumstances. E.g if you set a custom cache, it still sets this parameter, even if you don't use it.